emsdk

```diff
@@ -82,7 +82,7 @@ if not OSX and (platform.system() == 'Linux' or os.name == 'posix'):
 UNIX = (OSX or LINUX)
 
 ARCH = 'unknown'
-machine = platform.machine()
+machine = platform.machine().lower() #amd64 was not being recognised when returned as AMD64
```

Comment:
please add a space after the `#`

also please reword the comment for future readers (current comment looks good for readers of the patch now), like maybe `# lower() handles 'AMD64' which can happen in some cases`